lib/napa/setup.rb

```diff
@@ -32,5 +32,9 @@ def cache
     def cache=(store_option)
       @_cache = ActiveSupport::Cache.lookup_store(store_option)
     end
+    
+    def lookup_env
+      ENV['RACK_ENV'] || ENV['RAILS_ENV'] || 'development'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.